documentation/problem-forwarding-to-keptn.md

```diff
@@ -0,0 +1,76 @@
+# Forwarding problem notifications from Dynatrace to Keptn
+
+If you use Keptn to deploy your microservices and follow the standard tagging practices, Dynatrace will tag your monitored services with `keptn_project`, `keptn_service` and `keptn_stage`. If Dynatrace then detects a problem in one of these deployed services, e.g: High Failure Rate, Slow response time, you can let Dynatrace send these problems back to Keptn. To allow a Dynatrace problem to trigger a remediation workflow in Keptn, the dynatrace-service can intercept `sh.keptn.events.problem` events and forward the payload as a `sh.keptn.event.problem.open` event to the correct Keptn project, service and stage. All relevant problem details such as `PID`, `ProblemTitle` and `ProblemURL`are included.
```

Comment:
*and forward the payload as a `sh.keptn.event.problem.open` event to the correct Keptn project, service and stage.* 

This fact is not correct anymore. Please update it to: 

*and triggers a `remediation` sequence by sending a `sh.keptn.event.[STAGE_NAME].remediation.triggered` event for the correct Keptn project, service, and stage.*